universum_log_collapser/e2e/static_html.test.js

```diff
@@ -1,92 +1,34 @@
+const utils = require("./utils");
+
 beforeAll(async () => {
-    const path = require('path');
-    const currDir = path.dirname(module.filename);
-    await page.goto('file://' + currDir + '/test.html');
-    await waitForHandledResults(2);
+    await utils.openPage("test.html");
+    await utils.waitForHandledResults(2);
```

Comment:
It would be nice to declare a constant somewhere, describing that '2' is a current count of failed steps in sample output, and use it instead of simply passing magical '2' in every function usage. Using bare number is not only confusing, but is less convenient in case of changes to sample output